server-engine.go

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) 2012-2016 The Revel Framework Authors, All rights reserved.
```

Comment:
`serverengine.go` to follow the golang runtime convention of no hyphens or capitals?